app/assets/stylesheets/shared/_fonts.scss

```diff
@@ -1 +1,2 @@
 @import "https://fonts.googleapis.com/css?family=Open+Sans:300,700&subset=latin-ext";
+@import "https://fonts.googleapis.com/css?family=Cookie";
```

Comment:
Load both fonts in one request like so:
@import "https://fonts.googleapis.com/css?family=Open+Sans:300,700&subset=latin-ext;Cookie&text=Wizlist";